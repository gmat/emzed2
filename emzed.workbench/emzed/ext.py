import pkg_resources

for ep in pkg_resources.iter_entry_points("emzed_plugin", name="package"):
    print ep
    package = ep.load()
    exec "%s=package" % package.__name__
    del package
    del ep
del pkg_resources

