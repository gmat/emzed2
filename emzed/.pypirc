[distutils]
index-servers = 
  local

[local]
repository: http://127.0.0.1:8080
username:uschmitt
password:pillepalle

