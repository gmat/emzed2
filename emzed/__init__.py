# encoding:latin-1

from version import version as __version__


