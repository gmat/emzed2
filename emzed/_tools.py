
def is_started_from_cmdline():

    import inspect
    info = inspect.stack()[-1][1]
    result = False
    if info != "<string>":
        if info.endswith("emzed.workbench"):
            result = True
    return result


def gui_running():
    from PyQt4.QtGui import QApplication
    return QApplication.instance() is not None


def is_first_start():
    from emzed.core.config import _UserConfig
    import os
    return not os.path.exists(_UserConfig.config_file_path())
