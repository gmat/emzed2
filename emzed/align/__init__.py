from rt_align import rtAlign
from mz_align import mzAlign
