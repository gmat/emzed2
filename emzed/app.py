
import pkg_resources
reload(pkg_resources)  # migh be loaded already and is not up to date

for ep in pkg_resources.iter_entry_points("emzed_package", name="main"):
    runner = ep.load()
    print "$$$ loaded", runner
    exec "%s=runner" % ep.module_name.split(".")[0]
    del ep
    del runner

del pkg_resources

