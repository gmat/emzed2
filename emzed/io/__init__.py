
from load_utils import loadCSV, loadTable, loadPeakMap
from store_utils import storeCSV, storeTable, storePeakMap

try:
    del store_utils
except:
    pass
try:
    del load_utils
except:
    pass
