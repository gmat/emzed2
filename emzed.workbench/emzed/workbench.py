def run():
    print 42

def main():
    import os
    print os.system("pip install -i http://127.0.0.1:3142/root/dev/+simple emzed.core")
    import emzed.core
    print emzed.core



