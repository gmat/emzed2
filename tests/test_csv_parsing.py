import emzed.io
import os.path

def test_csv_parsing():
    here = os.path.dirname(os.path.abspath(__file__))
    tab = emzed.io.loadCSV(os.path.join(here, "data", "mass.csv"))
    #fms = '"%.2fm" % (o/60.0)'
    assert tab.getFormat("RT_min") == "%.2f"
