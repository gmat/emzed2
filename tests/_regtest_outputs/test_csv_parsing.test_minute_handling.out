rt      
float   
------  
0.50m   
3.00m   

rt
0.50m
3.00m

